content/client-server-api/_index.md

```diff
@@ -2831,7 +2831,38 @@ re-invited.
 
 {{% http-api spec="client-server" api="banning" %}}
 
-### Listing rooms
+### Room directory
+
+Homeservers MAY publish a room directory to allow users to discover rooms. A room
+can have one of two visibility settings in the directory:
+
+`public`
+The room will be shown in the published room directory.
+
+`private`
+The room will be hidden from the published room directory.
```

Comment:
```suggestion
-   `public`: The room will be shown in the published room directory.
-   `private`: The room will be hidden from the published room directory.
```

nit: since this appeared to fundamentally be a list.